Makefile

```diff
@@ -37,6 +37,7 @@ dist:
 	$(VIRTUALENV)/bin/python3 setup.py sdist bdist_wheel
 	aws s3 sync dist/ s3://datalabs-packages/wellcomeml
 	aws s3 cp --recursive --acl public-read dist/ s3://datalabs-public/wellcomeml
+	chmod +x create_release.sh && ./create_release.sh
```

Comment:
do you need to make it executable every time? does it work without it?

Reply:
I removed this